e2e/pod.go

```diff
@@ -105,17 +105,49 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	return nil
 }
 
-func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+func findContainerName(f *framework.Framework, c, ns, cn string, opt *metav1.ListOptions) (string, string, error) {
+	pn := ""
+	podList, err := f.PodClientNS(ns).List(context.TODO(), *opt)
+	framework.ExpectNoError(err)
+	if len(podList.Items) == 0 {
+		return "", "", errors.New("podlist is empty")
+	}
+	found := false
+	if cn != "" {
+		for i := range podList.Items {
```

Comment:
found is only using if the `cn` is not empty can we move found:=false inside if check?